frontends/web/src/locales/en/app.json

```diff
@@ -833,13 +833,33 @@
       }
     },
     "settings-electrum": {
+      "connection": {
+        "text": "If you  intend to only connect to your node when you are on the same network (e.g. your home wifi), then using regular network communication is sufficient.\nBe aware that your Electrum server needs to provide a TLS certificate to encrypt the communication in that case.\nIf you intend to connect to your node from anywhere, using Tor is the better option. No TLS certificate is necessary in that case.",
+        "title": "Should I use clearnet or Tor?"
+      },
+      "instructions": {
+        "link": {
+          "text": "Guide to connect your node",
+          "url": "https://guides.shiftcrypto.ch/bitboxapp/full-node/"
+        },
+        "text": "For a full tutorial, please visit our guide:",
+        "title": "How do I connect my BitBoxApp to my own fullnode?"
+      },
+      "options": {
+        "text": "There are multiple options to run your own node such as buying a finished device, building your own or running Bitcoin Core.\nIf you want to connect your BitBoxApp to your node, make sure that it runs an Electrum server. This is a dedicated program that allows a wallet apps to communicate with your full node.\nSupported options include Electrs, Electrum Personal Server (EPS) or Bitcoin Wallet Tracker (BTW).",
+        "title": "What options are there to run a node?"
+      },
+      "tor": {
+        "text": "Tor stands for 'The Onion Router', which is a free and open source software that offers  a lot of privacy benefits and is especially useful when using Bitcoin.\nIf you intend to connect to your node via Tor, make sure that Tor is installed on your computer and then enable the Tor Proxy in the BitBoxApp settings.\nOn most operating systems there are two ways to run Tor:\n1. Tor Browser: download and open the Tor Browser. This will allow the BitBoxApp to connect to the Tor network by setting port 9150 in the Tor proxy settings.\n2. Tor background service: install the Tor daemon, which always run in the background. The BitBoxApp can then connect by setting port 9050 in the Tor proxy settings.",
```

Comment:
a lot of text, maybe also better to make a dedicated guide and link it from here?


there is a double space.
@Stadicus mentioned sth about port 9051.

Reply:
This text was already reviewed by Stadicus before I made the PR. I think it's good ;)

Reply:
Not sure about the length, need to see that in the app. Ports: Tor uses 9050 and 9150. The xx51 ports are the "control ports", not relevant in this case.

Reply:
stilll double space

Reply:
removed whitespace